targets/TARGET_ARM_FM/TARGET_FVP_MPS2/us_ticker.c

```diff
@@ -77,16 +101,19 @@ void us_ticker_fire_interrupt(void)
 
 void us_ticker_disable_interrupt(void)
 {
-
-    US_TICKER_TIMER1->TimerControl &= 0xDF;
-    US_TICKER_TIMER2->TimerControl &= 0xDF;
-
+    US_TICKER_TIMER2->TimerControl &= ~CMSDK_DUALTIMER2_CTRL_INTEN_Msk;
```

Comment:
I think we should also disable ticker interrupt, not only TIMER2:
`NVIC_DisableIRQ(US_TICKER_TIMER_IRQn);`

Reply:
NVIC_DisableIRQ() added